themes/src/main/resources/theme/keycloak-preview/account/resources/app/content/applications-page/ApplicationsPage.tsx

```diff
@@ -113,27 +117,27 @@ export class ApplicationsPage extends React.Component<ApplicationsPageProps, App
             const appUrl: string = application.userConsentRequired ? application.baseUrl : '/auth' + application.baseUrl;
 
             return (
-              <DataListItem key={'application-' + appIndex} aria-labelledby="applications-list" isExpanded={this.state.isRowOpen[appIndex]}>
+              <DataListItem id={this.elementId("client-id", application)} key={'application-' + appIndex} aria-labelledby="applications-list" isExpanded={this.state.isRowOpen[appIndex]}>
                 <DataListItemRow>
                   <DataListToggle
                     onClick={() => this.onToggle(appIndex)}
                     isExpanded={this.state.isRowOpen[appIndex]}
-                    id={'applicationToggle' + appIndex}
+                    id={this.elementId('toggle', application)}
                     aria-controls="expandable"
```

Comment:
One last thing. This (`aria-controls`) should have the same value as this:
https://github.com/keycloak/keycloak/blob/ab862c045231dffaba342b400cf301d409aedc47/themes/src/main/resources/theme/keycloak-preview/account/resources/app/content/applications-page/ApplicationsPage.tsx#L152
I think this has something to do with accessibility. Even though we'll address accessibility in general later, let's not add more issues. ;)